package.json

```diff
@@ -125,6 +125,166 @@
+                                "additionalProperties": false,
+                                "type": "object"
+                            },
+                            "invokeTarget": {
+                                "title": "Invoke Target Properties",
+                                "description": "%AWS.configuration.description.awssam.debug.invokeTarget%",
+                                "properties": {
+                                    "lambdaHandler": {
+                                        "description": "%AWS.configuration.description.awssam.debug.lambdaHandler%",
+                                        "type": "string"
+                                    },
+                                    "projectRoot": {
+                                        "description": "%AWS.configuration.description.awssam.debug.projectRoot%",
+                                        "type": "string"
+                                    },
+                                    "samTemplatePath": {
+                                        "description": "%AWS.configuration.description.awssam.debug.samTemplatePath%",
+                                        "type": "string"
+                                    },
+                                    "samTemplateResource": {
+                                        "description": "%AWS.configuration.description.awssam.debug.samTemplateResource%",
+                                        "type": "string"
+                                    },
+                                    "target": {
+                                        "description": "%AWS.configuration.description.awssam.debug.target%",
+                                        "type": "string",
+                                        "enum": [
+                                            "code",
+                                            "template"
+                                        ]
+                                    }
+                                },
+                                "additionalProperties": false,
+                                "required": [
+                                    "target"
+                                ],
+                                "type": "object"
+                            },
+                            "lambda": {
+                                "title": "Lambda Properties",
+                                "description": "%AWS.configuration.description.awssam.debug.lambda%",
+                                "properties": {
+                                    "environmentVariables": {
+                                        "description": "%AWS.configuration.description.awssam.debug.envvars%",
+                                        "$ref": "#/definitions/jsonObject"
+                                    },
+                                    "event": {
+                                        "description": "%AWS.configuration.description.awssam.debug.event%",
```

Comment:
Missing a title

Reply:
I was more or less trying to mimic the previous structure - which did not have an interface for this bad-boy. Strong feelings that it should?

Reply:
Yeah, I'd say add it for consistency. Assume it was an oversight in the original.